keycloak/keycloak-admin-ui
# Adds functionality to attributes tab of realm roles section

## Motivation

https://issues.redhat.com/browse/APPDUX-563

## Brief Description

Adds functionality for realm role attributes tab. Can save `key, value` attribute pairs to server and these pairs remain visible in the UI. Can also delete `key, value` pairs from UI and server.

## Verification Steps



1. Go to Realm Roles section.
2. Create a new realm role.
3. Click on the "Attributes" tab.
4. Add a new key, value pair. 
5. Add another pair.
6. Try deleting a pair. 
7. Save.
8. Go back to the realm roles list. 
9. Click on the same role you just saved, and navigate back to the "Attributes" tab. 
10. Verify that the save function works correctly and all the attributes you added are shown in the UI. 


## Checklist:


## Additional Notes
